INIT: Using found mnist data
INIT: Finished creating train dataset
   label  score
i
0      2    NaN
1      0    NaN
2      4    NaN
3      8    NaN
6      0    NaN
MAIN: Started Training
Batch =0
2022-04-05 10:36:45.687329: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: UNKNOWN ERROR (34)
2022-04-05 10:36:45.687408: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (69dc0e526fe3): /proc/driver/nvidia/version does not exist
2022-04-05 10:36:45.687708: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Batch =100
Batch =200
Batch =300
Batch =400
Batch =500
Batch =600
Batch =700
Batch =800
Batch =900
Batch =1000
Batch =1100
Batch =1200
Batch =1300
Batch =1400
Batch =1500
Batch =1600
Batch =1700
/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py:6982: FutureWarning: In a future version, the Index constructor will not infer numeric dtypes when passed object-dtype sequences (matching Series behavior)
  return Index(sequences[0], name=names)
Traceback (most recent call last):
  File "RunTest.py", line 182, in <module>
    main(args)
  File "RunTest.py", line 140, in main
    df_train_losses = scoring_func(df_train_losses) # =(i|score,0,1,2...)
NameError: name 'scoring_func' is not defined